src/components/logs/Spinner.tsx

```diff
@@ -1,63 +1,42 @@
-import { CircularProgress } from '@mui/material';
+import { AlertColor } from '@mui/material';
+import { useMemo } from 'react';
 // import { MutableRefObject, useRef, useState } from 'react';
 import { useIntl } from 'react-intl';
 import { useLogsContext } from './Context';
 // import { useInterval } from 'react-use';
-import LogLine, { lineNumberColor } from './Line';
-
-// TODO (Spinner) Leaving the custom animation stuff in for right now. Still want to use this
-//     spinner but not worth the peformance cost compared to a simple css animated spinner
-
-// Spinner from https://github.com/sindresorhus/cli-spinners/blob/main/spinners.json
-// const ANIMATION_PROPERTIES = {
-//  interval: 150,
-//  frames: ['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏'],
-//  done: '⠿',
-// };
-
-// const getFrame = (frame: number) => {
-//  return ANIMATION_PROPERTIES.frames[frame];
-// };
+import LogLine from './Line';
+import SpinnerIcon from './SpinnerIcon';
+import { SpinnerMessageKeys } from './types';
+
+interface Props {
+    severity?: AlertColor;
+    runningKey?: SpinnerMessageKeys['runningKey'];
+    stoppedKey?: SpinnerMessageKeys['stoppedKey'];
+}
```

Comment:
Settings so far are to color the indicator, what to display when running/stopped. I considered adding in support for changing the loading indicator completely and an error/success setting but that felt overboard.